rdf/rdf12/rdf-n-triples/syntax/ntriples-base-1.nt

```diff
@@ -0,0 +1 @@
+<http://example/a>  <http://example/b> "Hello"@en--ltr .
```

Comment:
Minor nit: there are two spaces between `<http://example/a>` and `<http://example/b>`.
Not wrong, but a single space might be a bit nicer.

Reply:
Done.